pkg/fleets/fleets.go

```diff
@@ -44,10 +44,10 @@ func ListGameServerSetsByFleetOwner(gameServerSetLister listerv1.GameServerSetLi
 
 // ListGameServersByFleetOwner lists all GameServers that belong to a fleet through the
 // GameServer -> GameServerSet -> Fleet owner chain
-func ListGameServersByFleetOwner(gameServerLister listerv1.GameServerLister,
+func ListGameServersByFleetOwner(gameServerNamespacedLister listerv1.GameServerNamespaceLister,
```

Comment:
We most probably want to do the same for the `ListGameServerSetsByFleetOwner()`, it seems to be expected to work by namespace, but it's not the case (for the `syncFleet()` and `updateFleet()`